Fix consistency in tasks descriptions and namespaces usage

andreslucena opened the pull request:
#### :tophat: What? Why?

While working on #12676, I found some inconsistencies in the tasks:

1. We weren't consistents in the usage of the `upgrade` namespace, as there were some tasks that should have be ran only one time but didn't have this namespace 
2. Some descriptions weren't using the present correctly (i.e. saying "Check" instead of "Checks")
3. There was also two typo in a filename (decdim_proposal_upgrade_tasks.rake should be decidim_proposals_upgrade_tasks.rake)

This PR fixes it

#### Testing

Running `bin/rails -T |grep decidim` should have consistent verb tenses.
All the upgrade tasks (i.e. that should be run only one time during an update) should have the namespace 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!

andreslucena commented:
Opening this one as a first semi-draft. I have some doubts regarding other tasks that should be namespaced to `upgrade`: 

- `decidim:content_blocks:initialize_default_content_blocks`: it's only useful for the v0.28 and should not be run anymore 
- `decidim:procfile`:install: same, although it's also used in `./decidim-generators/lib/decidim/generators/install_generator.rb` 
- `decidim:robots:replace`: same as before, it's also used in `install_generator.rb`
- `decidim:webpacker:upgrade`: should be decidim:upgrade:webpacker
- `decidim_comments:update_participatory_process_in_comments`: should be `decidim_comments:upgrade:update_participatory_process_in_comments`

Does this make sense? We should also add a quick note in the releases notes just in case someone is upgrading from v0.27.x to v0.28.1 directly if this is backported

alecslupu reviewed (changes requested):
One small change, and i would be ok with this :)

alecslupu reviewed (dismissed):
:+1:

andreslucena commented:
I went ahead and applied some of these: 

> Opening this one as a first semi-draft. I have some doubts regarding other tasks that should be namespaced to `upgrade`:
> 
>     * `decidim:content_blocks:initialize_default_content_blocks`: it's only useful for the v0.28 and should not be run anymore
> 

Renamed to `decidim:upgrade:content_blocks:initialize_default_content_blocks` 

>     * `decidim:procfile`:install: same, although it's also used in `./decidim-generators/lib/decidim/generators/install_generator.rb`
> 
>     * `decidim:robots:replace`: same as before, it's also used in `install_generator.rb`
> 

These two were weird as they sounded (`decidim:upgrade:procfile:install` ??) so I've left them as they are. 

>     * `decidim:webpacker:upgrade`: should be decidim:upgrade:webpacker
> 

Renamed to `decidim:upgrade:webpacker`. I've left it in the same file as there were lots of methods reused with the install task 

>     * `decidim_comments:update_participatory_process_in_comments`: should be `decidim_comments:upgrade:update_participatory_process_in_comments`
> 

Renamed to `decidim_comments:upgrade:participatory_process_in_comments` 

I've also noticed that some of the upgrade modules namespaces werent consistent with the other modules: 

- `decidim:initiatives:upgrade:fix_broken_pages` renamed to `decidim_initiatives:upgrade:fix_broken_pages`
- `decidim:proposals:upgrade:remove_valuator_orphan_records` renamed to `decidim_proposals:upgrade:remove_valuator_orphan_records` 

This is ready for another round

andreslucena commented:
Changing it to `type: internal`, as I prefer to not backport this one, because we would need to add a notice for the content blocks migration task, as that's in the Releases notes for v0.28.0, and we could have installations that go from v0.27.x to v0.28.1 and the tasks would be different.

alecslupu reviewed (approved):
:+1:
